File: Exercicios_Em_Python_CV/desafio_014.py
#Escreva um programa que converta uma temperatura digitada temperatura
#em °C e converta para °F
temperatura_C = float(input('Digite a temperatura atual:'))
conversor_F = (9*temperatura_C+160)/5
print("temperatura em °C:{}",temperatura_C)
print("Temperatura em °F:{}",conversor_F)

File: Exercicios_Em_Java_CV/Desafio014.java
package fundamentos;

import java.util.Scanner;

/*
 * Escreva um programa que converta uma temperatura
 * digitada em °C e converta para °F
 */
public class Desafio014 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Digite a temperatura em °C:");
		float tempC = scanner.nextFloat();
		scanner.nextLine();
		System.out.printf("Nova temperatura em °F: %.2f", (tempC*9 + 160)/5);
		scanner.close();
	}

}
